docs/swagger-builder/readme.md

```diff
@@ -101,6 +101,3 @@ A code generator is provided to generate an extension of the **Amadeus Digital E
 ```bash
 yarn yo @ama-sdk/sdk:api-extension
```

Comment:
maybe remove reference to Amadeus Digital in the line above (line 99) ?

`A code generator is provided to generate an API specification extension.`